src/videouplinkbandwidthpolicy/VideoUplinkBandwidthPolicy.ts

```diff
@@ -57,4 +58,23 @@ export default interface VideoUplinkBandwidthPolicy {
    * Returns the selected [[MediaTrackConstraints]] to update
    */
   chooseMediaTrackConstraints(): MediaTrackConstraints;
+
+  /**
+   * Set a reference to the current transceiver controller.
+   * This should be called whenever a transceiver controller is available when the call starts.
+   * The default audio video controllers calls this at the end of connection.
+   * This method should not throw.
+   * @param {TransceiverController} transceiverController - The transceiver controller
+   */
+  setTransceiverController?(transceiverController: TransceiverController): void;
```

Comment:
Again, how do I know that I'm done with this controller (that is, when `updateTransceiverController` won't ever be called again)?

Why don't you need a `stop` method on this interface, or for this to be called with `undefined` when we're done?

Reply:
Maybe I am wrong but both the policy and transceiver should be cleaned up once the call is stopped because it is only referenced in our code in meetingContext object. So they only have to worry about it if they keep a reference to the policy on their side and if that is the case then they just need to remove that reference to the policy at the end right?

Reply:
It's not just about our code!

Reply:
The trick is: how does this object know "the end"?

Reply:
Updated to clean up the reference to transceiver controller at `CleanStoppedSessionTask` task